src/Fable.Transforms/Replacements.fs

```diff
@@ -2707,17 +2707,40 @@ let asyncs com (ctx: Context) r t (i: CallInfo) (_: Expr option) (args: Expr lis
     // Fable.Core extensions
     | meth -> Helper.CoreCall("Async", Naming.lowerFirst meth, t, args, i.SignatureArgTypes, ?loc=r) |> Some
 
-let guids (_: ICompiler) (ctx: Context) (_: SourceLocation option) t (i: CallInfo) (thisArg: Expr option) (args: Expr list) =
+let guids (com: ICompiler) (ctx: Context) (r: SourceLocation option) t (i: CallInfo) (thisArg: Expr option) (args: Expr list) =
+    let parseGuid (literalGuid: string) =
+        try
+            System.Guid.Parse(literalGuid) |> string |> makeStrConst |> Some
+        with e ->
+            e.Message |> addError com ctx.InlinePath r
+            None
```

Comment:
Returning None will trigger another error saying we couldn't find a replacement, which is not the case. You can use `addErrorAndReturnNull` instead.

Reply:
👍